config.yaml

```diff
@@ -127,35 +147,38 @@ params:
           - text: Documentation
             link: https://docs.pybamm.org/
 
-          - text: Citing PyBaMM
-            link: /publications/
-
-          - text: Ionworks
-            link: https://ion-works.com/
-
       column2:
         title: ""
         links:
           - text: Contact
             link: /contact/
 
           - text: Get involved
-            link: https://github.com/pybamm-team/PyBaMM/blob/develop/CONTRIBUTING.md/
+            link: /contribute/
 
           - text: Training
             link: /training/
 
+          - text: Ionworks
+            link: https://ion-works.com/
+
           - text: News
             link: /news/
 
-          - text: Roadmap
-            link: https://github.com/pybamm-team/PyBaMM/wiki/Roadmap/
+      column3:
+        title: ""
+        links:
+          - text: About PyBaMM
+            link: /about/
+
+          - text: Citing PyBaMM
+            link: /publications/
```

Comment:
Please let me know if the order for the footer links looks right. This was one of the combinations where all fourteen links fit nicely in a perfect grid haha